src/intl/en-US.js

```diff
@@ -368,6 +368,7 @@ export default {
   general: 'General',
   generalSettings: 'General settings',
   showSensitive: 'Show sensitive media by default',
+  showAllSpoilers: 'Always expand posts marked with content warnings',
```

Comment:
```suggestion
  showAllSpoilers: 'Expand content warnings by default',
```

To me, this fits better alongside "Show sensitive media by default".